app/apps/workflow/models.py

```diff
@@ -26,6 +28,16 @@
 logger = logging.getLogger(__name__)
 
 
+class CustomManager(models.Manager):
```

Comment:
Maybe add a comment on why we need this CustomManager. Maybe it can be separate from the workflow-app so it can be used by our other Django-apps as well. Seems like a util-class to me.

Reply:
Agreed